cypress/integration/tests/developer/applicationinventory/assessment/insecureRepo.test.ts

```diff
@@ -91,7 +91,7 @@ describe("Create an application form an insecure Git source", () => {
             getRandomApplicationData({
                 sourceData: {
                     repoType: "Subversion",
-                    sourceRepo: "http://github.com/main/test",
+                    sourceRepo: "http://github.com/konveyor/tackle-testapp",
```

Comment:
@midays This data is present in analysis.json under fixtures .
Please use it in this way 

https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/integration/tests/developer/applicationinventory/analysis/binary_analysis.test.ts#L93

Reply:
Done